execute/plugin_test.go

```diff
@@ -315,8 +316,8 @@ func Test_selectReport(t *testing.T) {
 			},
 			expectedExecReports:   1,
 			expectedCommitReports: 1,
-			expectedExecThings:    []int{5},
-			lastReportExecuted:    []cciptypes.SeqNum{100, 101, 102, 103, 104},
+			expectedExecThings:    []int{8},
+			lastReportExecuted:    []cciptypes.SeqNum{100, 101, 102, 103, 104, 105, 106, 107},
```

Comment:
Not sure why this would change, is it because it marshals to less JSON bytes?

Reply:
Yes exactly, this is potentially going to be a bit annoying until things settle down.

Could you fix this by adjusting `maxReportSize` instead of adjusting which messages were executed? While this still tests the same edge cases as before, that wont always be the case. Also it would keep the name "half report" accurate. Looks like you figured out that approach for most of the later tests.

Reply:
Yeah, that makes sense. I've fixed these to only tweak the sizes when needed.